cosi-provisioner/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This should be top level file.  Entire repo is ASL2.0

Reply:
removing it for now. we may need to have them later.